homeassistant/components/zha/sensor.py

```diff
@@ -486,6 +486,15 @@ def formatter(self, value: int) -> int:
         return round(pow(10, ((value - 1) / 10000)))
 
 
+class SmartEnergyMeteringClassMap(NamedTuple):
```

Comment:
Why don't we use entity descriptions for this? It's exactly the purpose they are designed for.

Reply:
Absolutely right, changed this